src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray.cs

```diff
@@ -145,7 +145,7 @@ public static ImmutableArray<T> CreateRange<T>(IEnumerable<T> items)
         [Pure]
         public static ImmutableArray<T> Create<T>(params T[]? items)
         {
-            if (items == null)
+            if (items == null || items.Length == 0)
             {
                 return Create<T>();
```

Comment:
Can this just return ` ImmutableArray<T>.Empty`?

Reply:
Yeah, I was thinking about that in the initial variant. I went for consistency, but I can change this, and if needed, all the others

Reply:
I think it would be a good idea to avoid the indirection via Create method.